docs/developer-docs/latest/development/typescript.md

```diff
@@ -132,3 +128,84 @@ const appContext = await strapi.compile();
 const app = await strapi(appContext);
 
 ```
+
+## Add TypeScript support to an existing Strapi project
+
+Adding TypeScript support to an existing project requires modifications to the project files. Additionally the TypeScript flag `allowJs` should be set to `true` in the root `tsconfig.json` file to incrementally add TypeScript files to existing JavaScript projects. The `allowJs` flag allows `.ts` and `.tsx` files to coexist with JavaScript files. TypeScript support can be added to an existing Strapi project using the following procedure:
```

Comment:
```suggestion
Adding TypeScript support to an existing project requires modifications to the project files. Additionally the TypeScript flag `allowJs` should be set to `true` in the root `tsconfig.json` file to incrementally add TypeScript files to existing JavaScript projects. The `allowJs` flag allows `.ts` and `.tsx` files to coexist with JavaScript files.

TypeScript support can be added to an existing Strapi project using the following procedure:
```

This is a bit more of a personal preference, but in other documentation sections, the introduction/concept paragraph is clearly separated from the "introductory" sentence for the procedure. 🤓